applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -620,28 +622,34 @@ gateway_to_endpoint(DestinationNumber
              ,{<<"Matched-Number">>, DestinationNumber}
               | gateway_from_uri_settings(Gateway)
              ]),
-    kz_json:from_list(
-      props:filter_empty(
-        [{<<"Route">>, gateway_dialstring(Gateway, DestinationNumber)}
-        ,{<<"Callee-ID-Name">>, kz_term:to_binary(DestinationNumber)}
-        ,{<<"Callee-ID-Number">>, kz_term:to_binary(DestinationNumber)}
-        ,{<<"To-DID">>, kz_term:to_binary(DestinationNumber)}
-        ,{<<"Invite-Format">>, InviteFormat}
-        ,{<<"Caller-ID-Type">>, CallerIdType}
-        ,{<<"Bypass-Media">>, BypassMedia}
-        ,{<<"Codecs">>, Codecs}
-        ,{<<"Auth-User">>, Username}
-        ,{<<"Auth-Password">>, Password}
-        ,{<<"Custom-SIP-Headers">>, SipHeaders}
-        ,{<<"SIP-Interface">>, SipInterface}
-        ,{<<"Endpoint-Type">>, EndpointType}
-        ,{<<"Endpoint-Options">>, EndpointOptions}
-        ,{<<"Endpoint-Progress-Timeout">>, kz_term:to_binary(ProgressTimeout)}
-        ,{<<"Custom-Channel-Vars">>, kz_json:from_list(CCVs)}
-        ,{<<"Outbound-Caller-ID-Number">>, CIDNumber}
-        ,{<<"Outbound-Caller-ID-Name">>, CIDName}
-         | maybe_get_t38(Gateway, OffnetJObj)
-        ])).
+    PropList = props:filter_empty(
+                 [{<<"Route">>, gateway_dialstring(Gateway, DestinationNumber)}
+                 ,{<<"Callee-ID-Name">>, kz_term:to_binary(DestinationNumber)}
+                 ,{<<"Callee-ID-Number">>, kz_term:to_binary(DestinationNumber)}
+                 ,{<<"To-DID">>, kz_term:to_binary(DestinationNumber)}
+                 ,{<<"Invite-Format">>, InviteFormat}
+                 ,{<<"Caller-ID-Type">>, CallerIdType}
+                 ,{<<"Bypass-Media">>, BypassMedia}
+                 ,{<<"Codecs">>, Codecs}
+                 ,{<<"Auth-User">>, Username}
+                 ,{<<"Auth-Password">>, Password}
+                 ,{<<"Custom-SIP-Headers">>, SipHeaders}
+                 ,{<<"SIP-Interface">>, SipInterface}
+                 ,{<<"Endpoint-Type">>, EndpointType}
+                 ,{<<"Endpoint-Options">>, EndpointOptions}
+                 ,{<<"Endpoint-Progress-Timeout">>, kz_term:to_binary(ProgressTimeout)}
+                 ,{<<"Custom-Channel-Vars">>, kz_json:from_list(CCVs)}
+                 ,{<<"Outbound-Caller-ID-Number">>, CIDNumber}
+                 ,{<<"Outbound-Caller-ID-Name">>, CIDName}
+                  | maybe_get_t38(Gateway, OffnetJObj)
+                 ]) ++ maybe_rtcp_mux(RTCP_MUX),
```

Comment:
you can remove the `props:filter_empty`. setting `RTCP-MUX` has no effect, it will not be delivered to ecallmgr because the amqp library will remove that header